telemetry/definitions/commonDefinitions.json

```diff
@@ -484,6 +494,31 @@
             "description": "Open to view status of an ECS Cluster",
             "metadata": [{ "type": "result" }]
         },
+        {
+            "name": "ecs_enableExecuteCommand",
+            "description": "Called when ECS execute command is enabled",
+            "metadata": [{ "type": "result" }]
+        },
+        {
+            "name": "ecs_disableExecuteCommand",
+            "description": "Called when ECS execute command is disabled",
+            "metadata": [{ "type": "result" }]
+        },
+        {
+            "name": "ecs_executeCommandSource",
+            "description": "Called when the option to run single commands in container or with a shell is chosen",
+            "metadata": [{ "type": "ecsExecuteCommandOrigin" }]
+        },
+        {
+            "name": "ecs_runExecuteCommand",
+            "description": "Called when the ECS execute command is run",
+            "metadata": [{ "type": "result" }]
```

Comment:
Could command source and shell selected be added to the metadata here instead of having separate metrics, or are those emitted for different events?

Reply:
Command Source could be added as a metadata here, but wanted to record it as another metric, separate from when the command is run.
Shell selected will be for emitted on a different event